id-repository/id-repository-core/src/test/resources/application-test.properties

```diff
@@ -363,6 +363,28 @@ logging.level.org.springframework=OFF
 logging.level.root=OFF
 spring.main.banner-mode=off
 
+mosip.idrepo.cache.names=credential_transaction,partner_extractor_formats,datashare_policies,topics,online_Verification_Partners,uin_encrypt_salt,uin_hash_salt,id_attributes
```

Comment:
Where is this used?

Reply:
This property is being used in simple cache config file sir.